src/main/java/org/broadinstitute/hellbender/engine/datasources/ReferenceMultiSource.java

```diff
@@ -49,8 +49,9 @@ public ReferenceMultiSource(final String referenceURL,
             } else {
                 referenceSource = new ReferenceFileSource(referenceURL);
```

Comment:
Completely unrelated question, but something that I realized reviewing this: is the separation of `ReferenceFileSource` and `ReferenceHadoopSource` really necessary? It looks like the `java.nio.Path` implementation would take care of Hadoop stuff (just for a different PR, but maybe worthy to open an issue about it)

Reply:
I think it's probably redundant like you say. We should take a look at it. #4279